docs/rebuild_admin.rst

```diff
@@ -0,0 +1,407 @@
+``./securedrop-admin install`` command to work correctly.
+
+Step 3: Set up Admin Workstation access
+=======================================
+Next, you'll configure the servers to allow temporary SSH access from the new *Admin
+Workstation*.
+
+First, start the new *Admin Workstation* with persistence enabled and an administration
+password set.
+
+Next, connect the new *Admin Workstation* to the *Hardware Firewall* via the
+appropriate Ethernet port, and set up its static IP address. For more information
+on how to do so, see :ref:`this section in the firewall setup documentation
+<assign_static_ip_to_workstation>`. If you do not know the correct static IP
+address for the *Admin Workstation*, log in to the *Hardware Firewall's* admin
+interface and check the settings under ``Aliases``.
+
+Next, determine whether your instance was set up to allow adminstrative access
+via SSH over Tor, or via SSH over LAN. If you don't know which option was originally
+chosen, you can check as follows:
+
+ #. Log in to the *Application Server* via the console using the adminstration username
+    and password.
+ #. Check to see if a file ``/var/lib/services/tor/ssh/hostname`` exists. If it
+    does, your instance is set up to use SSH over Tor and you should configure 
+    temporary SSH access using :ref:`these instructions <rebuild_ssh_over_tor>`.
+    If not, your instance is set up to use SSH over LAN, and you should follow
+    :ref:`these instructions instead <rebuild_ssh_over_lan>`.
+
+.. _rebuild_ssh_over_tor:
+
+Configuring access for an SSH-over-Tor instance
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Direct SSH access is disabled when the SSH-over-Tor option is selected during
+installation. To temporarily re-enable it, you'll need to update ``iptables`` rules
+and change the sshd daemon's configuration.
+
+First, log on to the *Application Server* via the console, and run the following
+commands, substituting the *Admin Workstation's* static IP for ``<admin_static_ip>``:
+
+.. code:: sh
+
+  sudo iptables -I INPUT -p tcp -dport 22 -s <admin_static_ip> \
+    -m state --state NEW,ESTABLISHED -j ACCEPT
+  sudo iptables -I OUTPUT -p tcp -sport 22 -m state --state ESTABLISHED -j ACCEPT 
+
+Next, edit the file ``/etc/ssh/sshd_config``, changing the line:
+
+.. code-block:: none
+
+  ListenAddress 127.0.0.1:22                                    
+
+to:
+ 
+.. code-block:: none
+
+  ListenAddress 0.0.0.0:22                                   
+
+and deleting the line: 
+
+.. code-block:: none
+
+  PasswordAuthentication no                                         
+
+Then, restart ``sshd`` using the command ``sudo service sshd restart``.
+
+Finally, check the file ``/etc/networking/interfaces``, and note down the network
+settings for the default Ethernet interface. You'll need them in the next step.
+
+Repeat the process above for the *Monitor Server*, making sure to note down its
+network settings as well.
+
+.. _rebuild_ssh_over_lan:
```

Comment:
I would recommend adding a note here to skip ahead to the section "Enabling access from the new *Admin Workstation*", since the next section does not apply to them, especially since we refer to "the next step" above.

Reply:
sounds good. added.